src/common_format.rs

```diff
@@ -173,6 +173,12 @@ pub fn manage_calendars<H>(file_handler: &mut H, collections: &mut Collections)
 where
     for<'a> &'a mut H: FileHandler,
 {
+    let calendar_exists =
+        file_handler.is_file("calendar_dates.txt") | file_handler.is_file("calendar.txt");
+    if !calendar_exists {
+        bail!("no calendar file found")
```

Comment:
this message could be better
`you should provide calendar.txt or calendar_dates.txt`
something like that

Reply:
It' now 
> calendar_dates.txt or calendar.txt not found
as in other checks of Navitia_model (`file {:?} not found`)